### PR TITLE
Set up project structure (and start work on major functions)

### LINKED ISSUE #2: Set up project structure
* Add the .Rpoj file & then add it to .gitignore 
* Create a the subdirectory folders such as `comparison_data`, `scripts`, `results` (in order to add to git there is going to need to be some sort of file in each directory if you don't have anything to add yet add a readme.md file that has a brief description of what the directory will eventually contain)

### LINKED ISSUE #4: optim related functions
Looking at the project plan develop the functions related to optim such as 
* get_co2_data 
* det_temp_data 
* objective_fn 
     * will probably want to add a test for this function 
* run_optim
* mse 
     * will probably want to add a test for this function 
*   get_var_mse

### PR DESCRIPTION
**Pull request by ptrscll**
This PR sets up the basic file structure outlined in the pseudocode document. It also includes the get_co2_data and get_temp_data functions, as well as a simple testing script I've been using to test functions as I write them

### CODE REVIEW

File: comparison_data/README.txt [hunk 1]
```diff
@@ -0,0 +1,11 @@
+CO2 data in co2_annmean_mlo.txt comes from NOAA data from Mauna Loa Observatory.
+
+Link to data: https://gml.noaa.gov/webdata/ccgg/trends/co2/co2_annmean_mlo.txt
+
+
+
+Temperature data in HadCRUT.5.0.2.0.analysis.summary_series.global.annual comes
+from the Met Office Hadley Centre observations datasets.
```
Comment (kdorheim) [inline comparison_data/README.txt]: Would recommend adding a note about the HadCRUT reference period

File: scripts/major_functions.R [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+#                 values (default: FALSE)
+#
+# returns: Hector-style data frame with temperature data
+get_temp_data <- function(file, scenario = "historical", include_unc = F) {
+  temp_data <- read.csv(file)
+  colnames(temp_data) <- c("year", "value", "lower", "upper")
+
+  # Adding in new columns to match Hector data frames
+  temp_data$scenario <- scenario
+  temp_data$variable <- GLOBAL_TAS()
```
Comment (kdorheim) [inline scripts/major_functions.R]: tas or gmst?

File: scripts/testing.R [hunk 3]
```diff
@@ -0,0 +1,29 @@
+# Script for testing functions
```
Comment (kdorheim) [inline scripts/testing.R]: Question for testing are you thinking about using this for the CI tests? Or is this kinda like a scratch pad that you are using while you are developing functions?

**Review by kdorheim — COMMENTED** [review]
Hi @ptrscll things are looking pretty good a few comments and questions for you. Could you link the open issues you are aiming to address in this PR? Do you want to keep working off of this one for a while?

File: scripts/major_functions.R (hunk 2 above)
Comment (ptrscll) [reply scripts/major_functions.R]: It should be tas, right? Isn't the HadCRUT data near-surface air temperatures? Or did I mix things up?

File: scripts/testing.R (hunk 3 above)
Comment (ptrscll) [reply scripts/testing.R]: So far, I've just been using this file like a scratch pad, but some stuff here might be useful for making CI tests.

**Comment by ptrscll** [comment]
Okay, at this point all the optim functions have been added and all of them except run_optim have been tested (either via unit tests or just the scratch file). I'm waiting to add the remaining table metrics functions until we resolve the issue with my TCR code, but once that's all set I think we should be ready to move on to running experiment one (which should also let us test run_optim).

**Review by kdorheim — APPROVED** [review]
@ptrscll this looks great! Please go ahead and merge this to main and then you can add the metric table code.

**Pull request merged**